inc/commondbrelation.class.php

```diff
@@ -1569,8 +1572,25 @@ protected static function getListForItemParams(CommonDBTM $item, $noent = false)
       $link = new $link_type;
```

Comment:
Still there is still cases where `$link_type` is not the name of a CommonDBTM class, we should throw an exception if `!is_a($link_type, CommonDBTM::class, true)`.